frontend/src/common/constants/routes.ts

```diff
@@ -9,4 +9,6 @@ export enum ROUTES {
   PREVIEW_POLICIES = "/previewpolicies/",
   WHERE_IS_MY_GENE = "/scExpression",
   DOCS = "/docs",
+  PUBLISHED_DATA_DOCS = "/docs/02__Find%20Published%20Data",
+  SCEXPRESSION_DOCS = "/docs/04__Analyze%20Public%20Data/4_2__scExpression%20Documentation/4_2_1__Get%20Started"
```

Comment:
nit: internally we call scexpression -> wmg

```suggestion
  WMG_DOCS = "/docs/04__Analyze%20Public%20Data/4_2__scExpression%20Documentation/4_2_1__Get%20Started"
```